letta/server/rest_api/routers/v1/sources.py

```diff
@@ -64,21 +64,22 @@ def list_sources(
     """
     actor = server.get_user_or_default(user_id=user_id)
 
-    return server.list_all_sources(user_id=actor.id)
+    return server.list_all_sources(actor=actor)
```

Comment:
shouldn't this be using the `source_manager`?

Reply:
no, there's extra logic tying it to agents